packages/text-annotator/src/state/TextAnnotationStore.ts

```diff
@@ -15,7 +15,9 @@ export interface TextAnnotationStore<T extends TextAnnotation = TextAnnotation>
 
   getAnnotationRects(id: string): DOMRect[];
 
-  getAt(x: number, y: number, filter?: Filter): T | undefined;
+  getAt(x: number, y: number, all: true, filter?: Filter): T[] | undefined;
+  getAt(x: number, y: number, all: false, filter?: Filter): T | undefined;
+  getAt(x: number, y: number, all?: boolean, filter?: Filter): T | T[] | undefined;
```

Comment:
What is the reason behind using explicit `true` or `false` values for the `all` argument? 🤔

Reply:
TS will automatically infer the return type correctly. (As either an annotation or an array.)

Reply:
Ah, that makes sense 👍🏻
I didn't notice the return type difference between those signatures